### DISCUSSION TITLE
I changed the board to rfsoc4x2 but encountered errors. Does anyone know why?

### DISCUSSION THREAD

**Discussion by harrisonasd** [Xilinx/finn#1011] [Q&A] [answered]
I changed the board to rfsoc4x2 but encountered errors. Does anyone know why?
![image](https://github.com/Xilinx/finn/assets/105301827/ea6c0d15-bd38-45c3-813d-dd572f2405ab)

![image](https://github.com/Xilinx/finn/assets/105301827/0927d504-9b66-4979-a76c-971d01e03cdd)

![image](https://github.com/Xilinx/finn/assets/105301827/6fcbd8d0-f15c-4193-9a3b-6703a5132b79)

![image](https://github.com/Xilinx/finn/assets/105301827/7e32deab-a05d-42c4-8a68-33d6ce0eb808)
Is it necessary to use the paid version of Vivado to use RFSoC?

**Comment by mariodruiz** [comment, marked as answer]
> Is it necessary to use the paid version of Vivado to use RFSoC?

Yes, you can find which devices need a **Vivado ML Enterprise Edition** here https://www.xilinx.com/products/design-tools/vivado/vivado-ml-buy.html#architecture

If you are an academic, you can request a Vivado license donation here: https://www.amd.com/en/corporate/university-program/donation-program.html

**Reply by harrisondusa** [reply]
I currently have a 30-day trial of Vivado ML Enterprise Edition, but I still encounter the same failed results

**Reply by mariodruiz** [reply]
Did you start the build process from scratch?

**Reply by harrisondusa** [reply]
I'm working on the 'tfc_end2end_example' from the 'bnn-pynq' notebook's 'end2end_example' and have executed it from start to finish once, but still encountered this error.

**Reply by harrisondusa** [reply]
![螢幕擷取畫面 (21)](https://github.com/Xilinx/finn/assets/142143716/766fe0d4-8878-4dab-a921-ba156495fc17)
I found that I haven't placed the license. Is this correct?

**Reply by fpjentzsch** [reply]
Yes, at least if $HOME is evaluated correctly and all paths are absolute. Although I'm not sure about the space before the colon.

**Comment by harrisondusa** [comment]
I currently have a 30-day trial of Vivado ML Enterprise Edition, but I still encounter the same failed results

**Comment by harrisondusa** [comment]
Has anyone successfully run RFSOC4x2 on FINN?

**Comment by harrisondusa** [comment]
![螢幕擷取畫面 (15)](https://github.com/Xilinx/finn/assets/142143716/5ff0c0ed-419e-4b3f-a1c9-7f37eba9553a)
![螢幕擷取畫面 (16)](https://github.com/Xilinx/finn/assets/142143716/65085647-96d2-40a2-aa56-063fe1079b3f)
![螢幕擷取畫面 (17)](https://github.com/Xilinx/finn/assets/142143716/7504265d-70ea-43a0-b043-8bf5b09b130d)

I found this error, but my license manager status is normal. Here are my screenshots.

**Comment by rafaruizortiz** [comment]
Hi @harrisonasd 

Did you solve the problem? I am having exactly the same output.

BTW, I used this same license in the past to synthesize projects for the RFSoC4x2 without FINN
